### PR TITLE
Feature/get in touch

### LINKED ISSUE #81: [SERVICES] Implement Support Services Page
## Description
The data services page implements both the normal page layout and data by priority section.

The order for this is laid out in the screenshots below.

## Screenshots

**Hero:**
screen shot 2018-06-25 at 16 51 08

**Data by Priority:**
screen shot 2018-06-14 at 15 57 59

**Get In Touch**
screen shot 2018-06-25 at 16 52 36

## Files

- Data Services : https://app.zeplin.io/project/5ae37a224a28f19738a929fb/screen/5b1a4341a2d56631598c8840
- Data by Priority: https://app.zeplin.io/project/5ae37a224a28f19738a929fb

- Data Services page: https://github.com/TakwimuAfrica/TAKWIMU/blob/develop/takwimu/templates/takwimu/about/data-services.html
- Data By Priority Section: ?

## Tasks

- [x] Style according to designs on Zeplin

### PR DESCRIPTION
**Pull request by mtuchi**
## Description

Add the services page and inquiry form in data services page.

Fixes #81 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
inq
serv

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by DavidLemayian** [comment]
@mtuchi Where does the content for the form go to?

**Comment by mtuchi** [comment]
@DavidLemayian 
Am not sure where it goes. But the referenced issue [zeplin designs](https://app.zeplin.io/project/5ae37a224a28f19738a929fb/screen/5b1a4341a2d56631598c8840) show's its part of the data services page. also there is a screenshot of the issue https://github.com/TakwimuAfrica/TAKWIMU/issues/81

**Comment by mtuchi** [comment]
@DavidLemayian 
I have implemented ` The service`  snippet in a laziest way possible so   it's just a duplicates of mediums articles

File: takwimu/templates/takwimu/_includes/sections/enquire.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                </div>
+            </div>
+            <div class="form-group">
+                <label for="help">How can we help you?</label>
+                <textarea class="form-control" id="help" rows="3"></textarea>
+            </div>
+            <button type="submit" class="btn btn-primary float-right">Submit Enquiry</button>
+        </form>
+    </div>
+</section>
```
Comment (KhadijaMahanga) [inline takwimu/templates/takwimu/_includes/sections/enquire.html]: Section needs to be in a `container`, and remove `col-8` for it first inner v

**Comment by mtuchi** [comment]
@DavidLemayian merged with recent changes

**Comment by DavidLemayian** [comment]
Thanks @mtuchi. Please update screenshots.

File: takwimu/templates/takwimu/_includes/sections/enquire.html (hunk 1 above)
Comment (DavidLemayian) [reply takwimu/templates/takwimu/_includes/sections/enquire.html]: cc @mtuchi

File: takwimu/templates/takwimu/about/support-services.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+{% include 'takwimu/_includes/sections/services.html' %}
+
+{% include 'takwimu/_includes/sections/enquire.html' %}
+
+
+{% include 'takwimu/_includes/partners.html' %}
+
+{% include 'takwimu/_includes/sections/updates.html' %}
+
+{% endblock %}
```
Comment (DavidLemayian) [inline takwimu/templates/takwimu/about/support-services.html]: Is this used anymore? I thought we are using a different template for the support services page?

File: takwimu/templates/takwimu/_includes/sections/services.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+                    <div class="card-body">
+                        <h5 class="card-title">{{ story.title }}</h5>
+                        <p class="card-text">{{ story.subtitle }}</p>
+                    </div>
+                </a>
+            </div>
+        </div>
+        {% endfor %}
+    </div>
+</section>
```
Comment (DavidLemayian) [inline takwimu/templates/takwimu/_includes/sections/services.html]: I think this has changed since this PR existed.

**Comment by mtuchi** [comment]
@DavidLemayian this PR needs some massive revamp, 
Event the contact us page link is not working, i will start working on it when am done with other stuff
Can you add this to v0.1.1 board if it needs a revamp

**Comment by mtuchi** [comment]
@kilemensi i think this was PR that fixed this issue 
https://github.com/TakwimuAfrica/TAKWIMU/pull/181

**Pull request closed without merging**